backend/infrastructures/apps/load/admin.py

```diff
@@ -0,0 +1 @@
+# Register your models here.
```

Comment:
An Django admin configuration should be added here.

Reply:
I added the config. please review and let me know, if it's what is expected.

Reply:
It just be adjusted to new model with JSONB field.